core/templates/dev/head/side_nav.html

```diff
@@ -8,11 +8,7 @@
     <li {% if nav_mode == 'library' %}class="active"{% endif %}>
       <a href="/library">
         <i class="material-icons oppia-sidebar-menu-icon">&#xE88A;</i>
-<<<<<<< HEAD
         <span translate="I18N_SIDEBAR_HOME_LINK"></span>
```

Comment:
Is this misnamed? Perhaps it should be I18N_SIDEBAR_LIBRARY_LINK, since the library is not the homepage anymore for signed-in users.